### PR TITLE
feat: email template with text

### PR DESCRIPTION
**Pull request by adithyanmadhi**
The basic html task to create a simple html email template
Screenshot 2024-01-05 172707

### CODE REVIEW

File: email-templates/task1.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <p style="color: #333;">
+    Dear Ronaldo,
+    
+  </p>
+
+  <p style="color: #333;">
+    Wishing you joy, success, and fulfillment in the coming year. May each day bring new opportunities and cherished moments. .
+  </p>
+
+  <p style="color: hsl(89, 51%, 46%);">
```
Comment (gkmahendran09) [inline email-templates/task1.html:21]: Instead of using `hsl` use hex values for colors. For example, for white use `#ffffff`

**Review by gkmahendran09 — CHANGES_REQUESTED** [review]
The commit messages should follow the conventional commits pattern. Some useful learning materials are.

- https://www.conventionalcommits.org/en/v1.0.0/
- https://gist.github.com/levibostian/71afa00ddc69688afebb215faab48fd7
- https://www.youtube.com/watch?v=OJqUWvmf4gg
- https://www.youtube.com/watch?v=JlfCRlFHmd8

File: email-templates/task1.html
[the remark this answers is not in the document]
Comment (adithyanmadhi) [reply email-templates/task1.html:1]: Updated the file name as email "template with text"

File: email-templates/task1.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>first email template</title>
```
[the remark this answers is not in the document]
Comment (adithyanmadhi) [reply email-templates/task1.html]: Used title case and updated.

File: email-templates/task1.html (hunk 1 above)
Comment (adithyanmadhi) [reply email-templates/task1.html:21]: Used hex values and updated

**Review by gkmahendran09 — APPROVED** [review]

**Pull request merged**